content/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md

```diff
@@ -223,32 +223,40 @@ Here are the metrics available to Prometheus:
 | redis_expire_cycle_cpu_milliseconds | The cumulative amount of time spent on active expiry cycles |
 | redis_expired_keys | Keys expired so far (since restart) |
 | redis_forwarding_state | Shard forwarding state (on or off) |
+| redis_keys_trimmed | The number of keys that were trimmed in the current or last resharding process |
 | redis_keyspace_read_hits | Rate of read operations accessing an existing keyspace (ops/sec) |
 | redis_keyspace_read_misses | Rate of read operations accessing an non-existing keyspace (ops/sec) |
 | redis_keyspace_write_hits | Rate of write operations accessing an existing keyspace (ops/sec) |
 | redis_keyspace_write_misses | Rate of write operations accessing an non-existing keyspace (ops/sec) |
```

Comment:
i now notice that the hits/misses are mentioning rate (ops/sec).
the metric is a "Running counter" like total_connections_received, redis_rdb_saves, redis_aof_rewrites, and aof_delayed_fsync (which i don't see here, please add)

Reply:
the `redis_aof_delayed_fsync` is a counter of how many times an AOF fsync caused delays in the redis main thread (inducing latency). it's an indication that the disk isn't fast enough or is overloaded.

Reply:
@oranagra And what is the aof_delayed_fsync?